paper.tex

```diff
@@ -107,11 +107,13 @@ \subsection{Data exchange mechanism for IoT}
+Researches below adopt MAM as data storage and a secure data transmission layer. The industrial data marketplace\cite{IOTAIdustryMarketplace} proposed by IOTA Foundation targets for IoT data streams trading. Service requesters call for proposal of interested data and accept/reject proposals from service providers. After a proposal is accepted, the requesters can access data streams stored via MAM and pay providers in IOTA tokens. This proposal matching and trading procedure can be automated without any human guidance\cite{IOTAIdustryMarketplaceWithoutHuman}. Nevertheless, this trading model does not take into account the refund and unsubscribe mechanism, the service requesters still need to pay even if they want to unsubscribe or the data does not meet expectations.
+
+O. Lamtzidis and J. Gialelis \cite{IOTASensorNode} proposed a distributed sensor node system to exchange data and establish a data monetization economy paradigm. The Back-End server helps sensor nodes to accelerate data uploads to MAM, manages keys and metadatas of data streams, and evaluates data price according to it's quality. Furthermore, the Back-End server operates an user friendly interface of the marketplace and tackles all trading procedures. The heavy loading of Back-End server encounters the system scalability problem, single point failure as well as malicious attacks which may damage the profits of all players. And the pricing strategy allows buyout, but does not take subscribe/unsubscribe based diagram into consideration. The decentralized data marketplace in \cite{DDMSmartCities} is fully decentralized which does not exist any intermediate server but data providers only, data providers attach and trade data streams via MAM. With MAM, the data streams subscription is enabled, yet the conditions and details like refund and unsubscription are not discussed in this paper. 
```

Comment:
should be "its quality."

Reply:
use "backend."

Reply:
The authors name the backend server `Back-End` server